FactorioBlueprintStringRenderer/src/com/demod/fbsr/BlueprintFinder.java

```diff
@@ -192,10 +196,10 @@ private static void findProviders(String content, CommandReporting reporting, Se
 				try {
 					String matchString = content.substring(matcher.start(), matcher.end());
 
-					System.out.println("\t[" + provider + "] " + matchString);
+					LOGGER.info("\t[{}] {}", provider, matchString);

```

Comment:
Most of these code replacements were performed by an auto-fix for an IntelliJ inspection that can replace concatenated System.out/err logging with LOGGER.info/error